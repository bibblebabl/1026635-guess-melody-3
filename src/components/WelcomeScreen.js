import React from 'react';
import PropTypes from 'prop-types';

const WelcomeScreen = ({errors}) => {
  return (
    <div>{errors}</div>
  );
};

WelcomeScreen.propTypes = {
  errors: PropTypes.string
};

export default WelcomeScreen;

