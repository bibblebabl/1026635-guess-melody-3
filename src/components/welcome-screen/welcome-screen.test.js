import React from 'react';
import renderer from 'react-test-renderer';
import WelcomeScreen from './welcome-screen';

const defaultProps = {
  errors: `some error`
};

it(`<WelcomeScreen /> renders correctly`, () => {
  const WelcomeScreenComponent = renderer.create(
      <WelcomeScreen {...defaultProps} />
  ).toJSON();

  expect(WelcomeScreenComponent).toMatchSnapshot();
});
