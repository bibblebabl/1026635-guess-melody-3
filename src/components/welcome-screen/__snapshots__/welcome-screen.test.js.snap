// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeScreen /> renders correctly 1`] = `
<div>
  some error
</div>
`;
